cloud/pkg/cloudhub/dispatcher/message_dispatcher.go

```diff
@@ -447,7 +451,7 @@ func noAckRequired(msg *beehivemodel.Message) bool {
 		return true
 	case msg.GetGroup() == modules.UserGroup:
 		return true
-	case msg.GetSource() == modules.NodeUpgradeJobControllerModuleName:
+	case msg.GetSource() == modules.NodeUpgradeJobControllerModuleName || msg.GetSource() == modules.ImagePrepullControllerModuleName:
```

Comment:
`NodeUpgradeJob` and `ImagePrepullJob` all should need ack? If needed, we can track in an issue.

Reply:
> `NodeUpgradeJob` and `ImagePrepullJob` all should need ack? If needed, we can track in an issue.

I have tried use ackMessage in imagePrepullJob, but I found Cloudhub's precession of ackmessage is not suitable for these operations. I will consider how to optimize it and track in an issue.